packages/vite/src/node/utils.ts

```diff
@@ -1656,3 +1656,17 @@ export const teardownSIGTERMListener = (
     }
   }
 }
+
+export function getServerUrlByHost(
+  server: PreviewServer | ViteDevServer,
+  host: CommonServerOptions['host'],
+): string | undefined {
+  if (host !== 'string') {
```

Comment:
```suggestion
  if (typeof host !== 'string') {
```
I think this is missing.

Reply:
Sorry 😂